### PR TITLE
todo list skeletal

### PR DESCRIPTION
**Pull request by Emmykage**
set up webpack
set up todo list dynamically 
linters check

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+}, {
+  index: '2',
+  list: 'take out the trash',
+  completed: false,
+
+},
+{
+  index: '3',
+  list: 'get some javascript done',
```
Comment (zieeco) [inline src/index.js]: **[REQUIRED CHANGES]**

- Please, let's try to follow the project requirement. replace the `list` with `description` to meet the project requirement below:

req

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            <input class= "checklist" type="checkbox" id="${todoList[i].index}">
+            <label for="${todoList[i].index}">${todoList[i].list}</label>
+            </div>
+           
+           <i class="del-btn fa-solid fa-ellipsis-vertical">...</i> 
+            `;
+
+  div.classList.add('listItems');
+  cont.appendChild(div);
+}
```
Comment (zieeco) [inline src/index.js]: **[REQUIRED CHANGES]**

- Please, let's try to follow the project requirement by placing this block of code in a function to meet the project requirement below:

func

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Nice design :heavy_check_mark: 
- Linters are passing :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
Hi,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**